mrp_operations_project/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+MRP Project Link (with operations)
+==================================
+
+This module links manufacturing operations with projects. It performs the
+following actions:
+
+* Auto-create tasks for work orders with assigned operators.
+* Add a tab where coding task works for the work order.
+* Add a link on task for the scheduled products for that work order.
+
+Installation
+============
+
+This modules is auto-installed when you install *mrp_operations_extension* and
+*mrp_project*.
+
+Usage
+=====
+
+In a manufacturing order (MO) that has work orders, and they have at least
+one operator assigned, when you start that work order, a task is created with
+that operator as responsible of it.
+
+Navigating through each work order, there's a new tab called "Operators time"
+where the task works can be put in an easy way.
+
+And last, if you go to a task linked to a work order, you'll see the scheduled
```

Comment:
"Besides, in the related tasks, the products scheduled in the work order are displayed."